### ISSUE TITLE
Aling to , support

### ISSUE THREAD

**Issue by sirsimon** [annsk/vscode-alignment#2]
```
array('FirstElem', 123, true);
array('SecondElem', 2, false);
```

align to:

```
array('FirstElem',  123, true);
array('SecondElem', 2,   false);
```

**Comment by martin-wiseweb** [comment]
Adding 
```
        "," : {
            "spaceAfter" : 1,
            "spaceBefore" : 0
        }
```

to your `alignment.chars` settings can allow you to achieve the following with the example code:

```
array('FirstElem' , 123, true);
array('SecondElem', 2  , false);
```

Not the same, but maybe a bit better than nothing?

**Comment by annsk** [comment]
I've added Align whitespace command.

Included in v0.3.0.
